docker/pipelines_runner/pipelines_runner.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -euo pipefail
+
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT_OPT="--project"
+TEMP_LOCATION_OPT="--temp_location"
+DOCKER_IMAGE_OPT="--docker_image"
+ZONES_OPT="--zones"
+COMMAND_OPT="--command"
+SERVICE_ACCOUNT_SCOPES="https://www.googleapis.com/auth/cloud-platform"
+
+#################################################
+# Parses arguments and does some sanity checking.
+# Arguments:
+#   It is expected that this is called with $@ of the main script.
+#################################################
+function parse_args {
+  while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+      ${PROJECT_OPT})
+        if [[ "$#" == 1 ]]; then
+          echo "ERROR: No project provided after $1!"
+          exit 1
+        fi
+        google_cloud_project="$2"
+        ;;
+
+      ${TEMP_LOCATION_OPT})
+        if [[ "$#" == 1 ]]; then
+          echo "ERROR: No temp location provided after $1!"
+          exit 1
+        fi
+        temp_location="$2"
+        ;;
+
+      ${DOCKER_IMAGE_OPT})
+        if [[ "$#" == 1 ]]; then
+          echo "ERROR: No docker image provided after $1!"
+          exit 1
+        fi
+        vt_docker_image="$2"
+        ;;
+
+      ${ZONES_OPT})
+        if [[ "$#" == 1 ]]; then
+          echo "ERROR: No zones provided after $1!"
+          exit 1
+        fi
+        zones="$2"
+        ;;
+
+      ${COMMAND_OPT})
```

Comment:
It might be nice to make this a positional argument instead

`docker run -v ~/.config:/root/.config ... vcf_to_bq --input_pattern INPUT_PATTERN --output_table OUTPUT_TABLE --temp_location TEMP_LOCATION`

Optional, though

Reply:
Done.